src/Components/Catalog/Poster/Poster.js

```diff
@@ -5,29 +5,19 @@ const Poster = ({films,loading}) => {
         return <h2>Loading...</h2>
     }
 
-
     return(
         <ul className={style.list}>
-            <li className={style.poster}>               
-                <img className={style.image} src = {films.Poster}/> 
-                <p className={style.title}>{films.Title}</p>
-                <p className={style.year}>{films.Year}</p>                         
-            </li>
-            <li className={style.poster}>               
-                <img className={style.image} src = {films.Poster}/> 
-                <p className={style.title}>{films.Title}</p>
-                <p className={style.year}>{films.Year}</p>                         
-            </li>
-            <li className={style.poster}>               
-                <img className={style.image} src = {films.Poster}/> 
-                <p className={style.title}>{films.Title}</p>
-                <p className={style.year}>{films.Year}</p>                         
-            </li>
-            <li className={style.poster}>               
-                <img className={style.image} src = {films.Poster}/> 
-                <p className={style.title}>{films.Title}</p>
-                <p className={style.year}>{films.Year}</p>                         
-            </li>
+            {
+                films.map((film,i) => (
+                    <li className={style.poster} key = {i}>
+                        <img className={style.image} src = {film.Poster}/> 
+                        <p className={style.title}>{film.Title}</p>
+                        <p className={style.year}>{film.Year}</p>
+                        <button className={style.info}>Info</button>
+                        <button className={style.remove}>Remove</button>
+                    </li>
```

Comment:
create separate component for the Film. And also add desctruction for the film. It is necessary to avoid film.Title and etc.